schema/cam_schema_1-1-2.json

```diff
@@ -120,7 +120,7 @@
                 },
                 "longitude": {
                   "type": "integer",
-                  "description": "Unit: 0,1 microdegree. oneMicrodegreeEast (10), oneMicrodegreeWest (-10), unavailable(1800000001)",
+                  "description": "Unit: 0.1 microdegree. oneMicrodegreeEast (10), oneMicrodegreeWest (-10), unavailable(1800000001)",
```

Comment:
We've got a a mix of 0.1 and 0,1 separator in all the files . Why just this?

Reply:
As discussed I'll open an issue about checking all schemas for commas to be replaced by dots